etc/regtest.bcoin.env

```diff
@@ -12,4 +12,4 @@ BCOIN_NO_WALLET=true
 BCOIN_INDEX_TX=true
 BCOIN_INDEX_ADDRESS=true
 BCOIN_NODE_HOST=0.0.0.0
-BCOIN_INIT_SCRIPT=funded-dummy-wallets.js
+BCOIN_INIT_SCRIPT=pagination-test-wallets.js
```

Comment:
I can do that - I guess the Q for you is how do you want to deal with this branch -- because it also targets the bcoin docker container with the pagination repo...

opts:
1) Leave as is, you and mark just grab the whole branch when you test
2) Make this a "real" PR with JUST the script and (like you say) comment out the other launch script (which by the way, still has that `consensus.coinbase_maturity` bug), then it can be merged but won't do any good without the pagination branch of bcoin
3) Make the bcoin docker target the `experimental` branch of bcoin (once that's a real thing) then we can merge, and then ALL bpanel development will be based around that dev branch of bcoin

Reply:
Option 3! ready to go with bpanel-org/bcoin#experimental :)

Reply:
hooray! done with latest commit